src/main/java/catan/controllers/GameController.java

```diff
@@ -95,8 +102,9 @@ public GameDetails getGameDetails(@RequestParam(value = "token", required = fals
                                       @RequestParam("gameId") String gameId) throws AuthenticationException, GameException {
         UserBean user = authenticationService.authenticateUserByToken(token);
         GameBean game = gameService.getGameByGameIdWithJoinedUser(user, gameId);
+        int userId = user.getId();
```

Comment:
remove this variable and use 'user.getId()' directly, less lines, less code, and code becommes more readable. And rename 'user' to something more familiar with "user that initiated game details"